src/components/Common/FollowButton.tsx

```diff
@@ -4,6 +4,9 @@ const styles = {
   followButton: {
     boxShadow: "none",
     backgroundColor: "primary.main",
+    textTransform: "none",
+    fontWeight: "bold",
+    minWidth: "84px",
```

Comment:
Minor nitpick, can we sort these style objects alphabetically? I wish we could have eslint checks for this

Reply:
sorted!